### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via ` codespell -q 3 -S ./thirdparty,*.po,./DONORS.md -L ackward,ang,ans,ba,beng,cas,childs,childrens,dof,doubleclick,expct,fave,findn,gird,hist,inout,leapyear,lod,nd,numer,ois,ony,paket,seeked,sinc,switchs,te,uint,varn`

### CODE REVIEW

File: scene/gui/graph_edit.cpp [hunk 1]
```diff
@@ -2107,7 +2107,7 @@ void GraphEdit::arrange_nodes() {
 		} while (u != E->get());
 	}
 
-	//Compute horizontal co-ordinates individually for layers to get uniform gap
+	//Compute horizontal coordinates individually for layers to get uniform gap
```
Comment (raulsntos) [inline scene/gui/graph_edit.cpp:2110]:
```suggestion
	// Compute horizontal coordinates individually for layers to get uniform gap.
```

File: scene/resources/animation.cpp [hunk 2]
```diff
@@ -4197,7 +4197,7 @@ struct AnimationCompressionDataState {
 	};
 
 	uint32_t components = 3;
-	LocalVector<uint8_t> data; //commited packets
+	LocalVector<uint8_t> data; //committed packets
```
Comment (raulsntos) [inline scene/resources/animation.cpp:4200]:
```suggestion
	LocalVector<uint8_t> data; // Committed packets.
```

File: scene/resources/animation.cpp [hunk 3]
```diff
@@ -4576,7 +4576,7 @@ void Animation::compress(uint32_t p_page_size, uint32_t p_fps, float p_split_tol
 				}
 			}
 			for (int j = 0; j < 3; j++) {
-				//cant have zero
+				//can't have zero
```
Comment (raulsntos) [inline scene/resources/animation.cpp:4579]:
```suggestion
				// Can't have zero.
```

File: scene/resources/animation.cpp [hunk 4]
```diff
@@ -4596,7 +4596,7 @@ void Animation::compress(uint32_t p_page_size, uint32_t p_fps, float p_split_tol
...
```
Comment (raulsntos) [inline scene/resources/animation.cpp:4599]:
```suggestion
				// Can't have zero.
```

File: scene/gui/rich_text_label.cpp [hunk 5]
```diff
@@ -4052,7 +4052,7 @@ int RichTextLabel::get_content_height() const {
 
 #ifndef DISABLE_DEPRECATED
 // People will be very angry, if their texts get erased, because of #39148. (3.x -> 4.0)
-// Altough some people may not used bbcode_text, so we only overwrite, if bbcode_text is not empty
+// Although some people may not used bbcode_text, so we only overwrite, if bbcode_text is not empty
```
Comment (raulsntos) [inline scene/gui/rich_text_label.cpp:4055]:
```suggestion
// Although some people may not used bbcode_text, so we only overwrite, if bbcode_text is not empty.
```

**Review by raulsntos — COMMENTED** [review]
Great work, just left some comments mostly nit-picks about fixing the comment style to use our current convention for the comments you are already changing anyway.

File: core/input/gamecontrollerdb.txt [hunk 6]
```diff
@@ -999,7 +999,7 @@ xinput,XInput Controller,a:b0,b:b1,back:b6,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,
...
 030000000d0f00006700000001010000,Horipad One,a:b0,b:b1,back:b6,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b8,leftshoulder:b4,leftstick:b9,lefttrigger:a2,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b10,righttrigger:a5,rightx:a3,righty:a4,start:b7,x:b2,y:b3,platform:Linux,
 030000000d0f0000c100000011010000,Horipad S,a:b1,b:b2,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b13,leftshoulder:b4,leftstick:b10,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b11,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b0,y:b3,platform:Linux,
-03000000341a000005f7000010010000,HuiJia GameCube Controller Adpater,a:b1,b:b2,dpdown:b14,dpleft:b15,dpright:b13,dpup:b12,lefttrigger:a3,leftx:a0,lefty:a1,rightshoulder:b7,righttrigger:a4,rightx:a5,righty:a2,start:b9,x:b0,y:b3,platform:Linux,
+03000000341a000005f7000010010000,HuiJia GameCube Controller Adapter,a:b1,b:b2,dpdown:b14,dpleft:b15,dpright:b13,dpup:b12,lefttrigger:a3,leftx:a0,lefty:a1,rightshoulder:b7,righttrigger:a4,rightx:a5,righty:a2,start:b9,x:b0,y:b3,platform:Linux,
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply core/input/gamecontrollerdb.txt:1002]: @raulsntos  May I have the repo address?

[the remark this answers is not in the document]
Comment (pycbouh) [reply core/input/gamecontrollerdb.txt:1002]: https://github.com/gabomdq/SDL_GameControllerDB

[the remark this answers is not in the document]
Comment (luzpaz) [reply core/input/gamecontrollerdb.txt:1002]: Submitted https://github.com/gabomdq/SDL_GameControllerDB/pull/531

[the remark this answers is not in the document]
Comment (luzpaz) [reply core/input/gamecontrollerdb.txt:1002]: Upstreamed merged changes in https://github.com/gabomdq/SDL_GameControllerDB/commit/a789be48bbea93d38ef6beb8177bb9d052e967d2

**Comment by mhilbrunner** [comment]
Thanks for submitting the fix for ` core/input/gamecontrollerdb.txt` upstream; please remove it from these changes for clarity.
Afterwards, the commits should be squashed into one commit.

All other changes look good to me.

File: core/input/gamecontrollerdb.txt (hunk 6 above)
[the remark this answers is not in the document]
Comment (luzpaz) [reply core/input/gamecontrollerdb.txt]: Removed change per https://github.com/godotengine/godot/pull/56315#issuecomment-1003651688

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you!

**Pull request merged**